scripts/configuration.py

```diff
@@ -19,23 +19,24 @@ class Configuration():
     full_screen  = True #Start application in full screen
     enable_email  = True #Enable the 'send email" feature
     enable_upload = True #Enable the upload feature
+    enable_print = True #Enable the printer feature
```

Comment:
Should be False by default (failproof)
Also, a `selected_printer` default value (of `None`?) is needed here